.gitignore

```
@@ -5,3 +5,7 @@
 # PhpUnit
 /.phpunit.result.cache
 /phpunit.xml
+
+# Tools
+/tools/*/composer.lock
+/tools/*/vendor
```

```suggestion
/tools/*/vendor
```

The composer.lock file should not be excluded, especially when the bundle is reusable.

---

See [09a11ec](https://github.com/contao/skeleton-bundle/pull/11/commits/09a11eccc04f15ca972c207eb94c50c0a8d0e406)